cloud-account/join-or-leave-an-organization.md

```diff
@@ -6,11 +6,30 @@ mapped_urls:
 
 # Join or leave an organization
 
-% What needs to be done: Refine
+## Accept an invitation [ec-accept-invitation]
 
-% Scope notes: https://github.com/elastic/docs-projects/issues/304
+Invitations to join an organization are sent by email. Invited users have 72 hours to accept the invitation. If they do not join within that period, an administrator of the organization will have to send a new invitation. Refer to [manage users](/deploy-manage/users-roles/cloud-organization/manage-users.md) for more information.
 
-% Use migrated content from existing pages that map to this page:
+## Leave an organization [ec-leave-organization]
 
-% - [ ] ./raw-migrated-files/cloud/cloud/ec-invite-users.md
-% - [ ] ./raw-migrated-files/docs-content/serverless/general-manage-organization.md
\ No newline at end of file
+On the **Members** tab of the **Organization** page, click the three dots corresponding to your email address and select **Leave organization**.
+
+If you’re the only user in the organization, you can only leave if you deleted all your deployments and projects, and you don’t have pending bills.
+
+## Join an organization from an existing {{ecloud}} account [ec-join-invitation]
+
+You already belong to an organization. If you want to join a new one and bring your deployments over, follow these steps:
+
+1. Backup your deployments to any private repository so that you can restore them to your new organization.
```

Comment:
do we have a topic we can link to to help them with this?

does this apply to serverless? if so, we need to make it more inclusive because they use projects instead of deployments

```suggestion
1. Back up your deployments to any private repository so that you can restore them to your new organization.
```